l10n/bundle.l10n.json

```
@@ -135,6 +135,9 @@
   "Choose": "Choose",
   "Choose and set default": "Choose and set default",
   "Do not load any": "Do not load any",
+  "Intellicode completions will not be provided, {0} failed to activate.": "Intellicode completions will not be provided, {0} failed to activate.",
```

```suggestion
  "IntelliCode features will not be available, {0} failed to activate.": "IntelliCode features will not be available, {0} failed to activate.",
```
Slightly tweaking this to both match the `IntelliCode` casing and be less specific about what features won't be available.

Theoretically we should be able to still provide starred completions even with the current problem, as those are two parallel features with no shared dependencies, but our activation code currently explodes if either fails.